clidocstool_md.go

```diff
@@ -179,8 +179,8 @@ func mdCmdOutput(cmd *cobra.Command, old string) (string, error) {
 
 	if cmd.Flags().HasAvailableFlags() {
 		fmt.Fprint(b, "### Options\n\n")
-		fmt.Fprint(b, "| Name | Description |\n")
-		fmt.Fprint(b, "| --- | --- |\n")
+		fmt.Fprint(b, "| Name | Type | Description |\n")
+		fmt.Fprint(b, "| --- | --- | --- |\n")
```

Comment:
Interesting; I thought that we already printed it in our reference docs, but looks like (for some reason) we're completely omitting it there https://docs.docker.com/engine/reference/commandline/run/)

Reply:
Ah we also have a `Default` column, didn't catch this one though.